metadata-integration/java/datahub-client/src/main/java/datahub/client/RestEmitter.java

```diff
@@ -19,36 +19,50 @@
 
 @Slf4j
 @RequiredArgsConstructor
-public class RESTEmitter {
+public class RestEmitter {
 
   private static final JacksonDataTemplateCodec DATA_TEMPLATE_CODEC = new JacksonDataTemplateCodec();
-
+  private static final int DEFAULT_CONNECT_TIMEOUT_SEC = 30;
+  private static final int DEFAULT_READ_TIMEOUT_SEC = 30;
+  
+  
   @Getter
   private final String gmsUrl;
```

Comment:
What is the format of this URL? We should also rename to be "serverUrl" because we are trying to move away from calling the Metadata Service "gms"